docs/paths/update_data_owner_op.yaml

```diff
@@ -0,0 +1,38 @@
+/updateDataOwner:
+  get:
+    tags:
+      - Data
+    summary: Update an owner of data
+    description: |
+                  Allows to update an owner of data
+
+                  #### Account Role Requirements
+                    | Account | Resource Type | Rule Action | Condition | Details |
+                    | --------|:-------------:|:-----------:| :----: |:-----: |
+                    | source  | `DATA`        | `UPDATE_OWNER`    | source `accountID` is equal to data owner | Resource contains data type |
+                    | source  | `DATA`        | `UPDATE_OWNER_FOR_OTHER`    | source `accountID` is **not** equal to data owner | Resource contains data type |
```

Comment:
- rule action should be named `TRANSFER_OWNERSHIP`
- we should not have `*_FOR_OTHER` action for this operation as it opens the probable attack vector
- add `TRANSFER_OWNERSHIP` to resource-account-rule.x and resource-signer-rule.x